packages/ui/src/components/Toaster/index.tsx

```diff
@@ -92,13 +82,17 @@ const CloseButton = ({ closeToast }: CloseButtonProps) => (
 
 type ContentProps = {
   children?: ReactNode
-  sentiment: 'danger' | 'info' | 'success'
+  sentiment:
+    | 'danger'
+    | 'success'
+    | 'warning'
+    | 'info' /** @deprecated Info is deprecated and will be removed in the next major update. */
 }
 
 const Content = ({ sentiment, children }: ContentProps) => (
   <Stack gap={2} direction="row">
     <Icon name={TOAST_ICONS[sentiment]} size={24} />
```

Comment:
You can remove this icon we don't have it anymore in the proto of design

<img width="358" alt="Screenshot 2024-02-12 at 13 29 10" src="https://github.com/scaleway/ultraviolet/assets/15812968/57cf7c91-e73e-4b8b-bd0e-22cd117caeea">